### DISCUSSION TITLE
Build Fails With No Build Logs

### DISCUSSION THREAD

**Discussion by DovahBrownies** [vercel/community#3872] [Help]
### Summary

I recently upgraded one of my very old projects from node `14.x` to `16.x`. I tried pushing the new changes to be deployed to Vercel but was consistently hit with a "Build Failed" error but with absolutely no error logs for me to diagnose.

![image](https://github.com/vercel/community/assets/35106956/e40fdaba-123c-4cb2-9fed-d732d70e0cc4)

I made sure to set the correct node version in the project settings:

![image](https://github.com/vercel/community/assets/35106956/efa23e93-472a-471c-b14a-07d01d9c26a2)

Here is a link to the failed deployment: https://vercel.com/dovah/portfolio/4HPvxnWuKPcE7atW2rHu4zbMLF6V

The closest similar discussion I've found is [this one](https://github.com/orgs/vercel/discussions/2950).
I've been struggling with this for 4 days trying to get this to work. Any help with this would be greatly appreciated!

Thank you in advance!

### Steps to Reproduce

- I push my commit to Github to start the Vercel deployment job
- Job fails without error

**Comment by amyegan** [comment]
Hey @DovahBrownies. I do see build logs available for that deployment. Can you try using a different browser or network, if possible?

**Reply by DovahBrownies** [reply]
I'm now on a completely different computer and I've just tried it on Chrome and Firefox and still no logs.

I also tried connecting my computer to my phone's mobile hotspot and still no logs.

![image](https://github.com/vercel/community/assets/35106956/4a4f521b-e246-4385-86b4-f0d55024085c)

Do you have any other suggestions I could try?

**Reply by DovahBrownies** [reply]
Not sure if this is related but... If I click on `Logs` and then go back to `Deployment`:

![image](https://github.com/vercel/community/assets/35106956/56308ad4-51ec-44ae-8183-708c4daa9cfe)

I can see the following error in the browser console:

![image](https://github.com/vercel/community/assets/35106956/90196bc2-6a29-47ad-89f7-d4a69c798496)

**Comment by DovahBrownies** [comment]
@amyegan Sorry for the ping - But is there any other action I can take to resolve this issue?

**Reply by amyegan** [reply]
It could be related to antivirus software settings. You can find some more information and troubleshoot steps in this guide: https://vercel.com/guides/why-are-my-build-logs-loading-infinitely-and-not-showing-up
